File: DatingApp.API/Models/Product.cs
using System;
using System.Collections.Generic;

namespace DatingApp.API.Models
{
    public class Product
    {
        public int Id { get; set; }

        public string ProductName { get; set; }

        public string ProductCode { get; set; }

        public DateTime ReleaseDate { get; set; }

        public float Price { get; set; }

        public string StarRating { get; set; }

        public string ImageUrl { get; set; }

        public List<Tag> Tags { get; set; }
    }
}

File: InventoryApp.Angular/src/app/models/product.ts
import { Photo } from './photo';

/* Defines the product entity */
export interface Product {
  id: number;
  productName: string;
  productCode: string;
  tags?: any[];
  releaseDate: string;
  price: number;
  description: string;
  starRating: number;
  imageUrl: string;
  photos?: Photo[];
}

